sdk/src/test/scala/com/horizen/account/receipt/EthereumLogTest.scala

```diff
@@ -0,0 +1,60 @@
+package com.horizen.account.receipt
+
+import com.horizen.account.receipt.EthereumLogTest.createTestEthereumConsensusDataLog
+import com.horizen.evm.utils.{Address, Hash}
+import com.horizen.utils.BytesUtils
+import org.junit.Assert._
+import org.junit._
+import org.scalatestplus.junit.JUnitSuite
+import org.scalatestplus.mockito._
+import scorex.crypto.hash.Keccak256
+
+class EthereumLogTest
+  extends JUnitSuite
+    with MockitoSugar
+{
+
+  @Test def receiptSimpleEncodeDecodeTest(): Unit = {
+    val ethereumLog = createTestEthereumConsensusDataLog
+    println(ethereumLog)
+    val encodedLog = EthereumConsensusDataLog.rlpEncode(ethereumLog)
+    println(BytesUtils.toHexString(encodedLog))
+    // read what you write
+    val dataBytes = encodedLog
+    val decodedConsensusDataLog = EthereumConsensusDataLog.rlpDecode(dataBytes)
+    System.out.println(decodedConsensusDataLog)
+    assertEquals(
+      BytesUtils.toHexString(ethereumLog.address.toBytes),
+      BytesUtils.toHexString(decodedConsensusDataLog.address.toBytes))
+  }
+
+}
+
+object EthereumLogTest {
```

Comment:
let's put it as a trait into `Fixtures` package

Reply:
Ok, done